### ISSUE TITLE
Failed to activate the git-rebase package

### ISSUE THREAD

**Issue by frese** [walles/git-rebase#1]
[Enter steps to reproduce:]

1. Failed during install !

**Atom**: 1.17.2 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.12.5
**Thrown From**: [git-rebase](https://github.com/walles/git-rebase) package 0.1.1

### Stack Trace

Failed to activate the git-rebase package

```
At _atom.CompositeDisposable is not a constructor

TypeError: _atom.CompositeDisposable is not a constructor
    at Object.activate (/packages/git-rebase/lib/git-rebase.js:9:26)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app/src/package.js:253:1)
    at /Applications/Atom.app/Contents/Resources/app/src/package.js:1014:1
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:1)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:1)
    at /Applications/Atom.app/Contents/Resources/app/src/package-manager.js:655:1
    at Set.forEach (native)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:653:1)
    at /Applications/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/package-manager.js:519:1
    at exit (/Applications/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/package-manager.js:128:1)
    at triggerExitCallback (/Applications/Atom.app/Contents/Resources/app/src/buffered-process.js:338:1)
    at /Applications/Atom.app/Contents/Resources/app/src/buffered-process.js:351:1
    at Socket.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/buffered-process.js:220:1)
    at emitOne (events.js:101:20)
    at Socket.emit (events.js:188:7)
    at Pipe._handle.close [as _onclose] (net.js:493:12)
```

### Commands

```
     -0:53.7.0 core:confirm (input.hidden-input)
```

### Non-Core Packages

```
ample-light-syntax 0.2.0 
atom-ctags 5.1.0 
autocomplete-ctags 0.4.1 
busy-signal 1.4.3 
dark-flat-ui 1.2.4 
dead-simple-light-syntax 0.1.0 
duotone-dark-earth-syntax 2.1.0 
duotone-denim-syntax 0.4.3 
duotone-light-syntax 2.1.0 
editorconfig 2.2.2 
file-icons 2.1.7 
gaia-syntax 2.0.0 
git-blame 1.1.1 
git-control 0.9.0 
git-history 3.3.0 
git-plus 7.9.3 
git-rebase 0.1.1 
git-status 0.3.4 
git-time-machine 1.5.9 
github-atom-light-syntax 0.4.1 
goto 1.8.3 
gruvbox-plus-syntax 1.3.1 
intentions 1.1.2 
language-diff 0.7.0 
language-slim 0.5.0 
language-terraform 0.8.1 
lazy-light-syntax 0.1.1 
linter 2.1.4 
linter-markdown 4.0.1 
linter-python-pep8 0.2.0 
linter-ruby 1.2.5 
linter-shellcheck 1.4.4 
linter-slim 1.0.0 
linter-terraform-syntax 1.2.0 
linter-ui-default 1.6.0 
make-runner 1.1.15 
markdown-preview-plus 2.4.9 
multi-cursor 2.1.5 
pen-paper-coffee-syntax 0.14.9 
pretty-json 1.6.3 
project-plus 1.0.0 
ruby-slim 0.2.0 
script 3.15.0 
slim-dark-ui 0.12.0 
slim-language 2.2.0 
slime 2.1.0 
summertime-syntax 0.1.0 
valign 1.1.3 
vim-mode-plus 0.93.0 
```

**Comment by frese** [comment]
Installs after upgrade to Atom 1.18 ! :-)

**Comment by martinsb** [comment]
It's still not fixed in my case - Atom 1.18, git-rebase 0.1.1
Stack trace
```
TypeError: _atom.CompositeDisposable is not a constructor
    at Object.activate (/Users/martinsb/.atom/packages/git-rebase/lib/git-rebase.js:9:26)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app/src/package.js:253:25)
    at /Applications/Atom.app/Contents/Resources/app/src/package.js:1014:34
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at PackageManager.module.exports.PackageManager.triggerActivationHook (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:689:49)
    at Workspace.handleGrammarUsed (/Applications/Atom.app/Contents/Resources/app/src/workspace.js:1387:40)
    at TextEditor.module.exports.TextEditor.observeGrammar (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:628:13)
    at /Applications/Atom.app/Contents/Resources/app/src/workspace.js:590:202
    at /Applications/Atom.app/Contents/Resources/app/src/workspace.js:595:19
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at PaneContainer.didAddPaneItem (/Applications/Atom.app/Contents/Resources/app/src/pane-container.js:266:24)
    at Pane.module.exports.Pane.addItem (/Applications/Atom.app/Contents/Resources/app/src/pane.js:532:22)
    at Pane.module.exports.Pane.activateItem (/Applications/Atom.app/Contents/Resources/app/src/pane.js:468:20)
    at Workspace.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/workspace.js:1161:20)
    at next (<anonymous>)
    at step (/Applications/Atom.app/Contents/Resources/app/src/workspace.js:11:279)
```

**Comment by jbouse** [comment]
Still seeing it... Atom 1.23.1 x64, git-rebase 0.1.1

Stack trace
```
TypeError: _atom.CompositeDisposable is not a constructor
    at Object.activate (/Users/jbouse/.atom/packages/git-rebase/lib/git-rebase.js:9:26)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app/src/package.js:255:25)
    at /Applications/Atom.app/Contents/Resources/app/src/package.js:1068:34
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at PackageManager.triggerDeferredActivationHooks (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:717:40)
    at Promise.all.then (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:650:18)
```
